library/OrderCloud.Catalyst/Auth/OrderCloudUserAuth.cs

```diff
@@ -52,11 +53,11 @@ public class OrderCloudUserAuthHandler<TSettings> : AuthenticationHandler<OrderC
 				var token = GetTokenFromAuthHeader();
 
 				if (string.IsNullOrEmpty(token))
-					return AuthenticateResult.Fail("The OrderCloud bearer token was not provided in the Authorization header.");
```

Comment:
Why are we getting rid of AuthenticateResult.Fail? Isn't that provided by the auth framework?

Reply:
My goal here is to return same 401 response structure as the platform. AuthenticateResult.Fail() does return a 401, but it has an empty body. I think Ideally I'd like to keep AuthenticateResult.Fail() but somehow hook into its behavior to return the body I want. I wasn't sure how to do that though

Reply:
Ok I think thats ok if ultimately it does the same thing. Just wanted to make sure we weren't short circuiting some other behavior